### PR TITLE
Add in versioning

### PR DESCRIPTION
**Pull request by thomasyu888**
* Add in `challengeutils` package version
* Add in semantic versioning for `challengeutils`

### CODE REVIEW

**Comment by tthyer** [comment]
Please add a brief description to your PRs.

File: challengeutils/__main__.py [hunk 1]
```diff
@@ -137,6 +138,9 @@ def build_parser():
         default=synapseclient.client.CONFIG_FILE,
         help="credentials file")
 
+    parser.add_argument('-v', '--version', action='version',
+                        version='Synapse Client {}'.format(__version__))
```
Comment (tthyer) [inline challengeutils/__main__.py]: Aren't challengeutils and synapse client separate things? If you're adding versioning for the former in this PR, why does this argument say Synapse Client?

Comment (thomasyu888) [reply challengeutils/__main__.py]: Copied this argparse from `synapseclient`.  Thanks for the catch!

**Comment by kdaily** [comment]
See #79 for a first pass at a developer focused contributing guide, based on the Python client contributing guide.

**Pull request merged**